README.md

```diff
@@ -73,6 +76,179 @@ variable "aks_subscription_id" {} # provided by the Jenkins library, ADO users w
 ```
 
 
+## Access to databases
+
+Databases are restricted to access from specific subnets, these can be updated in the [cnp-database-subnet-whitelisting](https://github.com/hmcts/cnp-database-subnet-whitelisting)
```

Comment:
I think that is not correct now that we use private endpoints

Reply:
Good catch - updated